### PR TITLE
[WebPubSubForSocketIO] Update some event design and add more tests

### PR DESCRIPTION
**Pull request by zackliu**
In this PR, I update some event design. Mainly added `closeUnderlyingConnection` to `DisconnectSocketAction` to keep consistent with origional socketio. Also, I move `namespace` to an optional parameter. The default value is "/", which is also the default one in original socketio.

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO/src/Bindings/Output/BaseActions/AddConnectionsToGroupsAction.cs
Comment (xingsy97) [inline sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO/src/Bindings/Output/BaseActions/AddConnectionsToGroupsAction.cs:11]: This comment seems incorrect. Same for remove one.

File: sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO/tests/JobHostEndToEndTests.cs
Comment (xingsy97) [inline sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO/tests/JobHostEndToEndTests.cs:220]: `arg2` before `arg1`?

Comment (zackliu) [reply sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO/tests/JobHostEndToEndTests.cs:220]: It's intended for validating order

**Review by jsquire — COMMENTED** [review]
Initial review of the common package version update looks good.   Please ping me when you've got all other approvals and are ready to merge and I'll re-review and approve to unblock the merge.

**Review by vicancy — APPROVED** [review]

**Comment by zackliu** [comment]
@jsquire Could you help to re-review and merge the PR

**Comment by zackliu** [comment]
@jsquire Could you help to re-review and merge the PR

**Review by jsquire — APPROVED** [review]

**Pull request merged**
